cloud_functions/climateiq_merge_output_cf/main.py

```diff
@@ -1,7 +1,98 @@
-from cloudevents import http
+import collections
+import csv
+import re
+
+import flask
+from google.cloud import storage
 import functions_framework
 
 
-@functions_framework.cloud_event
-def merge_scenario_predictions(cloud_event: http.CloudEvent) -> None:
-    return
+# Bucket name, where prediction outputs are stored.
+BUCKET_NAME = "climateiq-predictions"
+# File name pattern for the CSVs for each scenario and chunk.
+CHUNK_FILE_NAME_PATTERN = (
+    r"(?P<run_id>\w+)/(?P<prediction_type>\w+)/(?P<model_id>\w+)/"
+    r"(?P<study_area_name>\w+)/(?P<scenario_id>\w+)/(?P<chunk_id>\w+)\.csv"
+)
+# Directory in bucket to write merged files.
+OUTPUT_DIR = "merged"
+
+
+# This only handles merging flood data. Heat data will be divided into completely
```

Comment:
Per Walt, the chunks for heat predictions will be per day (as opposed to flood, which are chunked by area). If we want to merge the heat and flood predictions together, it will take some additional transforms.

Reply:
we don't need to merge the heat and flood predictions together- just scenarios across both hazards. I confirmed this with Daniel/Jesus a while back

Reply:
^ lets remove this comment or reword it- its a bit confusing. rn all steps of the pipeline only handle flood predictions anyways